src/mlui/enums/Adabound.py

```diff
@@ -0,0 +1,225 @@
+
+
+
+import tensorflow.compat.v2 as tf
+from keras import backend
+from keras.optimizers import Optimizer
+
+class AdaBound(Optimizer):
+    """
+    Adabound optimizer implementation 
+    # References
+        - [Adaptive Gradient Methods with Dynamic Bound of Learning Rate]
+          (https://openreview.net/forum?id=Bkg3g2R9FX)
+
+
+
+    Parameters
+    -----------
+        learning_rate : float 
+            Adam learning rate (default: 1e-3)
+
+        base_lr: float >= 0.
+             Used for loading the optimizer. Do not set the argument manually.    
```

Comment:
The default value should be explicitly set as `None` here. There should be spaces on both sides of the semicolon `:`. There should be no dot at the end of the type annotation, only after the description.

Reply:
solved